test/unit_tests/js/modules/util/add-email-popup-spec.js

```diff
@@ -57,7 +55,6 @@ describe( 'add-email-popup', () => {
   } );
 
   it( 'should hide the email popup initially', () => {
-    const addEmailPopup = require( BASE_JS_PATH + '/add-email-popup' );
```

Comment:
This is removing the file that is being tested. With this change the spec won't be testing anything and the coverage will be 0% for that file.

Reply:
I fixed this lint warning with no-unused-vars by using eslint-disable-next-line before addEmailPopup